kingpin/actors/aws/ecs.py

```diff
@@ -406,7 +420,369 @@ def _execute(self):
         if self.option('wait'):
             yield self._wait_for_tasks(tasks)
         else:
-            self.log.info('Not waiting for tasks to stop')
+            self.log.info('Not waiting for tasks to complete')
+
+
+# http://boto3.readthedocs.io/en/latest/reference/services/ecs.html
+SERVICE_DEFINITION_SCHEMA = {
+    'type': 'object',
+    'additionalProperties': False,
```

Comment:
Are you sure? I was pretty sure that they just pass the JSOn right on up to the API and let the API deal with it.